project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject amazonica "0.3.21"
+(defproject amazonica "0.3.22-SNAPSHOT"
```

Comment:
I've never used snapshots with this project, as I push most changes like this to Clojars.